src/views/Admin/Modules/Records/List.vue

```diff
@@ -114,5 +121,28 @@ export default {
       rowCreateUrl: 'admin.modules.record.create',
     }
   },
+
+  methods: {
+    ...mapActions({
+      updateModule: 'module/update',
+    }),
+
+    handleSave (fields = []) {
+      fields = JSON.parse(JSON.stringify(fields))
+      if (!this.module.meta.ui) {
+        this.module.meta.ui = {
+          admin: {
+            fields,
+          },
+        }
+      } else {
+        this.module.meta.ui.admin = { fields }
```

Comment:
This works now when we have only fields under admin, but if we had something else it would be overwritten